cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
Volumes are not necessarily where the puck is headed. I still have real questions around dev/prod parity with volumes many of which are apparent by the files "lost" by mounting over them with volumes. Those files would suddenly reappear in production.